wireapp/kalium
# feat: handle conversation member change event (AR-2337)

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?
There was an unhandled event for updating member role changes. 

### Solutions

Handle WebSocket event and update member role in local DB

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

Update member role in conversation and check if it updates in other device conversation details screen

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like  images, videos, etc. (drag and drop in the text box)_

https://user-images.githubusercontent.com/13151239/190134167-89083665-d0c0-40da-8b8d-cbfd57c9adc1.mov


----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
